### PR TITLE
add crono systemd scripts

### LINKED ISSUE #261: Create systemd unit file for portus-cronos
We recently started to use `cronos` to handle some background jobs. We have to ensure the cronos service is started automatically inside of the appliance.

I think the cleanest solution is to create a systemd unit file for that.

Note well: cronos must be started in foreground mode otherwise systemd will have troubles handling it.

@jordimassaguerpla: I can help you with that if you want

### PR DESCRIPTION
**Pull request by jordimassaguerpla**
fixes #261

Signed-off-by: Jordi Massaguer Pla <jmassaguerpla@suse.de>

### CODE REVIEW

File: packaging/suse/conf/portus_crono.service [hunk 1]
```diff
@@ -0,0 +1,11 @@
+[Unit]
+Description=A time-based background job scheduler daemon
+After=apache2.service
```
Comment (flavio) [inline packaging/suse/conf/portus_crono.service]: I don't think it required apache2 to be running. @mssola does the code executed by crono talk with Portus via its API? I think it goes straight to the DB. If that's the case we would need to have mariadb up and running, not apache.

**Comment by flavio** [comment]
LGTM except for the comment I made. We can ignore Travis' results, they are not related with these changes.

File: packaging/suse/conf/portus_crono.service (hunk 1 above)
Comment (jordimassaguerpla) [inline packaging/suse/conf/portus_crono.service]: mariadb can be in another server thus I would instead propose to have network running.

Comment (mssola) [inline packaging/suse/conf/portus_crono.service]: Apache is indeed not needed. We only need the DB.

**Pull request merged**